R/tokens.R

```diff
@@ -248,9 +248,8 @@ tokens.corpus <- function(x,
     if ("remove_twitter" %in% names(dots)) {
         .Defunct(msg = "'remove_twitter' is defunct, use 'what = \"word2\"' instead.")
     }
-    unused_dots(dots)
+    check_dots(dots, c(names(formals(tokens))))
```

Comment:
I changed all the `check_dots()` to simpler `unused_dots()` in all the functions. We should choose either.

Reply:
unused_dots() is fine, but I needed to be able to change (...) in the parent function (for trapping deprecated arguments) and then send the resulting (modified) list to the check_dots(). You cannot use list(...) in the child fn if it's already been evaluated - then it's a list one level deep. So just to get this branch working, for now, I added back the previous fn. But you are right we only should use one.